test/cookbooks/dokken_test/recipes/default.rb

```diff
@@ -1,23 +1,38 @@
-docker_service 'default' do
-  host ['tcp://127.0.0.1']
-  action [:create, :start]
-end
-
 user 'notroot' do
   home '/home/notroot'
   manage_home true
   action :create
 end
 
-package_list = %w(git ruby ruby-devel rubygem-io-console rubygem-bundler rubygem-rake gcc redhat-rpm-config libffi libffi-devel)
+package_list = %w(
+  gcc
+  git
+  iputils
+  libffi
+  libffi-devel
+  net-tools
+  nmap
+  procps-ng
+  redhat-rpm-config
+  ruby
+  ruby-devel
+  rubygem-bundler
+  rubygem-io-console
+  rubygem-rake
+  telnet
+  which
+)
 
-package package_list do
-  action :install
+package package_list
+
+docker_service 'default' do
+  host ['tcp://127.0.0.1']
+  action [:create, :start]
 end
 
 git '/home/notroot/kitchen-dokken' do
-  repository 'https://github.com/someara/kitchen-dokken'
-  revision 'master'
+  repository '/opt/kitchen-dokken/.git'
+  revision node['dokken_test']['revision']
```

Comment:
Needed a way for Travis to build the gem from the PR commit instead of from the master branch.